frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/print/elements/NodePrintElement.java

```diff
@@ -0,0 +1,45 @@
+
+package com.virtuslab.gitmachete.frontend.graph.print.elements;
+
+import com.intellij.vcs.log.graph.api.elements.GraphElement;
+import org.checkerframework.checker.index.qual.NonNegative;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import com.virtuslab.gitmachete.frontend.graph.GraphElementManager;
+
+public class NodePrintElement extends PrintElementWithGraphElement implements INodePrintElement {
+
+  public NodePrintElement(@NonNegative int rowIndex,
+      @NonNegative int positionInCurrentRow,
+      GraphElement graphElement,
+      GraphElementManager graphElementManager) {
+    super(rowIndex, positionInCurrentRow, graphElement, graphElementManager);
+  }
+
+  @Override
+  public boolean equals(@Nullable Object o) {
```

Comment:
Ditto, maybe also specify contract for equality in the interface

Reply:
Actually the depending on the default implementation is enough